Dockerfile

```diff
@@ -60,6 +60,9 @@ RUN pip install https://github.com/pfultz2/rclone/archive/master.tar.gz
 # Install yapf
 RUN pip3 install yapf==0.28.0
 
+# Install numpy
+RUN pip3 install numpy
```

Comment:
I dont see numpy used anywhere.

Reply:
I tried to use numpy to create a scalar input for testing and added it here. But it still does not work. So changed to use others. Is it  the correct way to add numpy to be used in the above unit test?

Reply:
So can this be removed? Or whats the plan?

Reply:
I removed this